modules/basicUI.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en-us">
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width" />
+  <title>CSS basic UI module sample</title>
+  <style>
+body {
+  font-family: sans-serif;
+  font-size: 1.25rem;
+}
+[contenteditable] {
+  cursor: copy;
+  caret-color: magenta;
+  border: 1px solid #ccc;
+}
+:focus {
+  outline: dashed magenta 3px;
+  outline-offset: 10px;
+}
+* {
+  accent-color: magenta;
+}
+div, fieldset {
+  margin: 20px;
+}
+textarea:nth-of-type(1) {
+  cursor: wait;
+}
+textarea:nth-of-type(2) {
+  resize: none;
+}
+textarea:nth-of-type(3) {
+  pointer-events: none;
+}
+  </style>
+</head>
+<body>
+<div><span contenteditable>Edit this text </span></div>
+<fieldset><legend>Play with these fake form controls</legend>
+    <input type="checkbox" id="check">
+    <input type="radio" name="a">
+    <input type="radio" name="a">
+    <input type="range">
+    <progress>
```

Comment:
I think this is the right way of doing it:
```suggestion
    <progress></progress>
```